demo/index.html

```diff
@@ -107,6 +107,7 @@
   <cat-carousel
```

Comment:
Don't replace existing demo cause this demo is used for normal mode
I Recommend you duplicate this code and create new one with center mode under existing demo

Reply:
I need to make sure also the existing production code isn't breaking so I think you need to create new one with your mode

Reply:
Got it